test/Integration/tpp-run-splat-memref.mlir

```diff
@@ -35,7 +35,7 @@ func.func @entry(%input: memref<4x2xf32>) {
 // RANDOM-SPLAT-NOT: @__wrapper_0 : memref<4x2xf32> = dense<1.000000e+00>
 // RANDOM-SPLAT-NOT: constant @__constant_2x16xf32 : memref<2x16xf32> = dense<1.000000e+00>
 // RANDOM-SPLAT-NOT: constant @__constant_4x16xf32 : memref<4x16xf32> = dense<2.000000e+00>
-// RANDOM-SPLAT-NOT: constant @__constant_4x8xf32 : memref<4x8xf32> = dense<0.000000e+00>
+// RANDOM-SPLAT: constant @__constant_4x8xf32 : memref<4x8xf32> = dense<0.000000e+00>
```

Comment:
Why wouldn't it get randomized?

Reply:
Because it may be a dense zero for ReLU (`generic(in, zeros) { maxf(%in, %zero) }`).

I'm also avoiding `dense<-1.0>` because that's used as filling for vector conversion.

Reply:
Right, I didn't think about that. Good catch.